src/Tribe/PUE/Checker.php

```diff
@@ -467,68 +467,50 @@ public function ajax_validate_key() {
 		/**
 		 * processes variable substitutions for server-side API message
 		 */
-		private function get_api_message( $info ) {
+		private function get_api_message( $pluginInfo ) {
 			// this default message should never show, but is here as a fallback just in case.
 			$message = sprintf(
 				esc_html__( 'Sorry, there is a problem with your license key. You\'ll need to %scheck your license%s to have access to updates, downloads, and support.', 'tribe-common' ),
 				'<a href="https://theeventscalendar.com/license-keys/">',
 				'</a>'
 			);
 
-			if ( ! empty( $info->api_invalid_message ) ) {
-				$message = wp_kses( $info->api_invalid_message, 'post' );
+			if ( ! empty( $pluginInfo->api_invalid_message ) ) {
+				$message = wp_kses( $pluginInfo->api_invalid_message, 'post' );
 			}
 
 			$message = str_replace( '%plugin_name%', '<b>' . $this->get_plugin_name() . '</b>', $message );
 			$message = str_replace( '%plugin_slug%', $this->get_slug(), $message );
 			$message = str_replace( '%update_url%', $this->get_pue_update_url(), $message );
-			$message = str_replace( '%version%', $info->version, $message );
+			$message = str_replace( '%version%', $pluginInfo->version, $message );
 
 			return $message;
 		}
 
 		/**
-		 * Echo JSON formatted errors
+		 * Displays an error notice if a premium plugin is activated and the license is expired
+		 *
+		 * @since 4.3
 		 */
-		public function display_json_error() {
-			$pluginInfo       = $this->json_error;
-			$update_dismissed = $this->get_option( $this->dismiss_upgrade );
-
-			$is_dismissed = ! empty( $update_dismissed ) && in_array( $pluginInfo->version, $update_dismissed ) ? true : false;
-
-			if ( $is_dismissed ) {
-				return;
-			}
+		public function display_expired_license_message() {
+			$pluginInfo  = $this->json_error;
 
 			if ( ! current_user_can( 'administrator' ) ) {
 				return;
 			}
+			$expired_license_msg     = $this->get_api_message( $pluginInfo );
+			$expired_license_message = str_replace( '%plugin_name%', '<strong>' . $this->get_plugin_name() . '</strong>', $expired_license_msg );
+			?>
+			<div class="notice notice-warning is-dismissible" id="pu-dashboard-message">
+				<div class="tribe-message">
+					<img class="spirit-animal" src="<?php esc_html_e( plugins_url( '../../src/resources/images/tec-panda.png', dirname( __FILE__ ) ) );?>">
```

Comment:
How about a class change, `esc_url`, and point at the file using the handy-dandy `plugin_url` property of `Tribe__Main`:

```
<img class="tribe-spirit-animal" src="<?php esc_url( Tribe__Main::instance()->plugin_url . 'common/src/resources/images/tec-panda.png' );?>">
```